.github/templates/linux_binary_build_workflow.yml.j2

```diff
@@ -71,12 +71,17 @@ jobs:
       {%- if config.pytorch_extra_install_requirements is defined and config.pytorch_extra_install_requirements|d('')|length > 0  %}
       PYTORCH_EXTRA_INSTALL_REQUIREMENTS: !{{ config.pytorch_extra_install_requirements }}
       {%- endif %}
+      {%- if config["gpu_arch_type"] == "cuda-aarch64" %}
+      timeout-minutes: 360
```

Comment:
@tinglvv Please create an issue about aarch64 gpu machine slow compile times. And mention here in comments

Reply:
Created https://github.com/pytorch/pytorch/issues/126980 to track.